standalone_self_upgrade.cf.in

```diff
@@ -673,11 +677,20 @@ bundle agent cfengine_master_software_content
        create => "true",
        comment => "We need a place to download each packge we build";
 
+      "$(download_dir)/$(cfengine_software.pkg_version)-downloaded.txt"
```

Comment:
what if one or two of all the binaries fail to download due to network issues or something? They won't be listed in downloaded.txt but I suppose the platform will fail when trying to download the binary package during self upgrade.

Reply:
Yeah, it's not great. Basically if any succeeds, then we consider it success so that the hub can avoid running the self upgrade sub-agent each execution. While that doesn't seem to cause any functional issue, it does result in a log file per update run on the hub, which is not ideal.